modules/planning/tasks/deciders/open_space_roi_decider.cc

```diff
@@ -46,6 +46,13 @@ OpenSpaceRoiDecider::OpenSpaceRoiDecider(const TaskConfig &config)
 Status OpenSpaceRoiDecider::Process(Frame *frame) {
   // TODO(Jinyun) only run GetOpenSpaceROI() at the first frame of
   // open space planner to save computation effort
+
+  if (frame == nullptr) {
+    const std::string msg =
+        "Invalid frame, fail to process the OpenSpaceRoiDecider.";
+    return Status(ErrorCode::PLANNING_ERROR, msg);
```

Comment:
Simply put the msg into the statement to reduce a local variable.

Reply:
Oh, I forgot to add an AERROR for the message.